src/java/org/apache/cassandra/exceptions/InvalidColumnTypeException.java

```diff
@@ -22,6 +22,7 @@
 
 import org.apache.cassandra.cql3.ColumnIdentifier;
 import org.apache.cassandra.db.marshal.AbstractType;
+import org.apache.cassandra.io.sstable.format.Version;
```

Comment:
Nit: unused

Reply:
I'll cleanup imports after squashing the commits